src/doc/trpl/arrays-vectors-and-slices.md

```diff
@@ -49,7 +49,7 @@ languages.
 
 A *vector* is a dynamic or "growable" array, implemented as the standard
 library type [`Vec<T>`](../std/vec/) (we'll talk about what the `<T>` means
-later). Vectors always allocate their data on the heap. Vectors are to slices 
+later). Vectors always allocate their data on the heap. Vectors are to arrays
```

Comment:
I pondered this twice on two separate passes and still decided my change was correct, so maybe that says something about ambiguity.

`&str` is stack-allocated, `String` is heap-allocated; arrays are stack-allocated, vectors are heap-allocated. The preceding chapter refers to `String`s as "growable", this paragraph refers to vectors as growable. That's how I'm reading the analogy. If this isn't right, I'd argue there's something nonintuitive here. IIRC there is is absolutely no other use of the term "slice" in the guides up to this point except for "string slices", so what else can "Vectors are to slices what `String` is to `&str`" possibly be referring to?